File: FizzBuzz/FizzBuzzGame/FizzBuzzGame/FizzBuzz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FizzBuzzGame
{
    public class FizzBuzz
    {
        public string ReturnValue(int value)
        {
            if (value == 3)
            {
                return "Fizz";
            }
            return value.ToString();
        }
    }
}

File: Javascript/Fizzbuzz/fizzbuzz.js
exports.returnValue = function (value) {
    if (isFizz(value) && isBuzz(value)) {
        return "FizzBuzz";
    }
    if (isFizz(value)) {
        return "Fizz";
    }
    if (isBuzz(value)) {
        return "Buzz";
    }
    return value;
}
function isFizz(value) {
    return value % 3 == 0;
}
function isBuzz(value) {
    return value % 5 == 0;
}